packages/xo-web/src/xo-app/host/tab-advanced.js

```diff
@@ -65,7 +65,7 @@ const SCHED_GRAN_TYPE_OPTIONS = [
 const forceReboot = host => restartHost(host, true)
 
 const smartReboot = ALLOW_SMART_REBOOT
-  ? host => restartHost(host, false, true) // don't force, suspend resident VMs
+  ? host => restartHost(host, false, true, false) // don't force, suspend resident VMs
```

Comment:
Quite unrelated to this PR but maybe we should switch to named parameters, it's getting [hard to read](https://www.30secondsofcode.org/js/s/boolean-trap/#boolean-trap-what-s-in-a-name). For now, at least update the comment that explains the meaning of each boolean argument.